### ISSUE TITLE
Training params of Mesh Decoder

### ISSUE THREAD

**Issue by zoujx96** [microsoft/TRELLIS#178]
Hi, Thanks for your great work! Could you provide your training configurations of the mesh decoder, like what's the batch size, learning rate, training iterations and how many views do you use per object? Also do you use mixed-precision training?
